enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/EnterpriseCoreEditionModule.java

```diff
@@ -264,6 +267,19 @@ public void registerEditionSpecificProcedures( Procedures procedures ) throws Ke
         life.add( coreServerModule.membershipWaiterLifecycle );
     }
 
+    private LogFiles buildLocalDatabaseLogFiles( PlatformModule platformModule, FileSystemAbstraction fileSystem,
+            File storeDir )
+    {
+        try
+        {
+            return LogFilesBuilder.activeFilesBuilder( storeDir, fileSystem, platformModule.pageCache ).withConfig( config ).build();
+        }
+        catch ( IOException e )
+        {
+            throw new RuntimeException( e );
```

Comment:
The caller throws `KernelException` right? Would it be better to transform the `IOException` into that instead?

Reply:
sorry not sure I get it, its invoked from the constructor of EnterpriseCoreEditionModule and it does not throw an exception.
what if I will create some custom exception for those two cases?

Reply:
Github shows diffs in mysterious ways, ignore this comment